File: app/src/main/java/marcosmiranda/seniasnicas/WordList.java
package marcosmiranda.seniasnicas;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;

import com.marcosmiranda.seniasnicas.R;

public class WordList extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_word_list);

        Intent intent = getIntent();
        String message = intent.getStringExtra(Diccionario.EXTRA_CATEGORY);
        TextView textView = findViewById(R.id.textView);
        textView.setText(message);
    }
}

File: app/src/main/java/com/marcosmiranda/seniasnicas/WordList.kt
package com.marcosmiranda.seniasnicas

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class WordList : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_word_list)
        val intent = intent
        val message = intent.getStringExtra(Diccionario.EXTRA_CATEGORY)
        val textView = findViewById<TextView>(R.id.textView)
        textView.text = message
    }
}
